README.md

```diff
@@ -2,10 +2,10 @@
 
 The adapter was initially developed for conjugate heat transfer (CHT) simulations via preCICE by Lucia Cheung in the scope of her master’s thesis [[1]](https://www5.in.tum.de/pub/Cheung2016_Thesis.pdf) in cooperation with [SimScale](https://www.simscale.com/). For running the adapter for CHT simulations refer to this thesis. The adapter was extended to fluid-structure interaction by Alexander Rusch [[2]](https://www.gacm2017.uni-stuttgart.de/registration/Upload/ExtendedAbstracts/ExtendedAbstract_0138.pdf).
 
-This adapter was developed for **CalculiX version 2.16.** Other versions may be compatible, yet they have not been tested. Please let us know if you want to use a different version.
+This adapter was developed for **CalculiX version 2.17.** Other versions may be compatible, yet they have not been tested. Please let us know if you want to use a different version.
```

Comment:
Must this not be for 2.19?

Reply:
Indeed, that's something that got merged from v2.17 and I forgot to update afterwards.

Reply:
Great, otherwise I am happy